test/integration/rememberable_test.rb

```diff
@@ -120,6 +120,40 @@ def cookie_expires(key)
     end
   end
 
+  test 'extends remember period when extend remember period config is true' do
+    swap Devise, extend_remember_period: true, remember_for: 1.year do
+      user = create_user_and_remember
+      old_remember_token = nil
+
+      travel_to 1.day.ago do
+        get root_path
+        old_remember_token = request.cookies['remember_user_token']
+      end
+
+      get root_path
+      current_remember_token = request.cookies['remember_user_token']
+
+      refute_equal old_remember_token, current_remember_token
```

Comment:
@josevalim My assertion was the cookie contents changing.

I could be more assertive checking the expiration keys, but the cookie is encrypted.

Worth decrypt and check the key? How I can do it?